Fakebook.Profile/Fakebook.Profile.RestApi/Controllers/ProfileController.cs

```diff
@@ -162,5 +162,46 @@ public async Task<ActionResult> UpdateAsync([FromBody] ProfileApiModel apiModel)
                 return BadRequest();
             }
         }
+
+        [HttpPost("follow/{user=user}")]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<ActionResult> Follow(ProfileApiModel user)
+        {
+            // Get emails of each user
+            string thisUserEmail = GetUserEmail();
+            string followEmail = user.Email;
+            if (thisUserEmail is null)
+            {
+                return NotFound(thisUserEmail);
+            }
+            if (followEmail is null)
+            {
+                return NotFound(followEmail);
```

Comment:
404 refers strictly to the resource identified by the URL. this would be a 400 instead, so you could use RequiredAttribute to let the framework handle it

Reply:
Changed to required attribute and BadRequest. Don't I only need to keep one of these changes?